client/src/GettingStarted/GettingStarted.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { Button, Card, Col, Row, Typography } from 'antd'
+import Icon from '@material-ui/core/Icon'
+import AssignmentIcon from '@material-ui/icons/Assignment'
+import BusinessIcon from '@material-ui/icons/Business'
+import CloudUploadIcon from '@material-ui/icons/CloudUpload'
+import PlaylistAddIcon from '@material-ui/icons/PlaylistAdd'
+
+export function GettingStarted() {
+  /*
+  TODO:
+  - layout/general
+    - ask about outer layout
+    - change classnames to suit P4P
+    - double-check responsive breakpoints
```

Comment:
If you look in `tailwind.config.js` we have our breakpoints laid out under "screens"

Reply:
Ah, I think what I was confused about was which responsive system to use. This component is using the antd `Row` and `Col` approach, and the breakpoints are currently expressed like
```
<Col xs={24} sm={12} lg={6}>
```
On further review, it seems like a fair amount of existing code uses the Tailwind style like
```
<div className="grid grid-cols-1 medium:grid-cols-8 large:grid-cols-2">
```
Would it be preferable for me to switch systems?

Reply:
Yeah, we're moving away from `Row` and `Col` because we want to use tailwind's layout classes instead.  Sorry I didn't catch that on first review!